angular-rails-app/config/database.yml

```diff
@@ -33,8 +33,10 @@ development:
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
 test:
-  <<: *default
-  database: heatseek_test
+  adapter: sqlite3
+  database: db/test.sqlite3
+  pool: 5
+  timeout: 5000
```

Comment:
Nice. This should be brought over to my branch.

I assume this automatically handles creating a new database if none already exists?